src/components/VLink.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <Component :is="linkComponent" v-bind="linkProperties" v-on="$listeners"
+    ><slot
+  /></Component>
+</template>
+
+<script>
+/**
+ * This is a wrapper component for all links. All VLinks have to have a valid `href` attr.
+ * Links with `href` starting with `/` are treated as internal links.
+ *
+ * Internal links use `NuxtLink` component with `to` attribute set to `localePath(href)`
+ * External links use `a` element set to open in a new tab and not raise an error with the current iframe setup.
+ */
+import { computed, defineComponent, useContext } from '@nuxtjs/composition-api'
+import { warn } from '@/utils/warn'
+const defaultProps = { target: '_blank', rel: 'noopener noreferrer' }
+export default defineComponent({
+  name: 'VLink',
+  setup(_, { attrs }) {
+    const { app } = useContext()
+    const isInternal = computed(() => attrs.href?.startsWith('/'))
+    const linkComponent = computed(() => (isInternal.value ? 'NuxtLink' : 'a'))
+    const { href, ...otherProperties } = attrs
```

Comment:
Destructuring these from `attrs` will make their values non-reactive in the `computed` for `linkProperties` below. To maintain reactivity they need to stay as properties on `props` or `attrs`.